buildcraft_resources/assets/buildcraft/lang/en_US.lang

```diff
@@ -478,7 +478,7 @@ itemGroup.buildcraft.plugs=BuildCraft Pluggables
 
 tile.architectBlock.name=Architect Table
 tile.assemblyTableBlock.name=Assembly Table
-tile.assemblyWorkbenchBlock.name=Advanced Crafting Table
+tile.advancedCraftingTableBlock.name=Advanced Crafting Table
```

Comment:
You misunderstand: we shouldn't change lang keys from 1.7.10 unless the thing that they represent has changed its name.

If the code is pointing to the wrong lang key then it should be fixed on the code side. (This saves us from having to change every lang file)